Tests/conf.json

```diff
@@ -4866,10 +4874,8 @@
         "EWS Extension: Powershell Online V2 Test": "Issue 39008",
         "O365 - EWS - Extension - Test": "Issue 39008",
         "Majestic Million Test Playbook": "Issue 30931",
-        "iDefense_v2_Test": "Issue 40126",
```

Comment:
I see [this issue](https://github.com/demisto/etc/issues/40126) is closed, can we un-ignore the TPB now? @kgal-pan